### PR TITLE
Add CI badges

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -13,6 +13,10 @@ finite automata and guarantees linear time matching on all inputs.
 """
 categories = ["text-processing"]
 
+[badges]
+travis-ci = { repository = "rust-lang/regex" }
+appveyor = { repository = "rust-lang-libs/regex" }
```
Comment (BurntSushi) [inline Cargo.toml]: Should this be `rust-lang/regex`?

Comment (mgeisler) [inline Cargo.toml]: I took the name from the CI badge that you already have in the README, and for some reason the name is different on AppVeyor compared to TravisCI.

Comment (BurntSushi) [inline Cargo.toml]: Interesting. YOLO.

**Comment by BurntSushi** [comment]
Thanks! @bors r+

File: Cargo.toml (hunk 1 above)
Comment (mgeisler) [inline Cargo.toml]: Hehe :-D

**Pull request merged**